src/replication.c

```diff
@@ -1432,47 +1432,27 @@ static int useDisklessLoad() {
 }
 
 /* Helper function for readSyncBulkPayload() to make backups of the current
- * DBs before socket-loading the new ones. The backups may be restored later
- * or freed by disklessLoadRestoreBackups(). */
-redisDb *disklessLoadMakeBackups(void) {
-    redisDb *backups = zmalloc(sizeof(redisDb)*server.dbnum);
-    for (int i=0; i<server.dbnum; i++) {
-        backups[i] = server.db[i];
-        server.db[i].dict = dictCreate(&dbDictType,NULL);
-        server.db[i].expires = dictCreate(&keyptrDictType,NULL);
-    }
-    return backups;
+ * DBs and cluster slots to keys map before socket-loading the new ones.
+ * The backups may be restored by disklessLoadRestoreBackup or freed by
+ * disklessLoadDiscardBackup later. */
+dbBackup *disklessLoadMakeBackup(void) {
+    return backupDb();
```

Comment:
i'm not certain we need these wrappers. i think that any new things we'll need to do when creating backups (like adding a module notification in a later PR this week), will be done in db.c, and these wrappers will remain thin wrappers forever.
But also, i guess they don't cause any harm, and they do add comments that better explain when and how they're used.

Reply:
I am not sure, comments are important to help us understand. Initially I think wrappers are easy to add some sample records or somethings if we want.

I am ok if you want to remove wrappers.

Reply:
let's keep them for now..

someone will probably make an "optimization" PR some day suggesting to remove them. 8-)